src/Timeline/styled.js

```diff
@@ -0,0 +1,40 @@
+import styled from 'reakit/styled';
+import { palette } from 'styled-tools';
+import Box from 'reakit/Box';
+
+export const Row = styled(Box)`
+  display: flex;
+  position: relative;
+
+  &:last-child {
+    margin-bottom: 0;
+  }
+
+  &::before {
+    content: '';
+    border: 1px solid ${props => props.lineColor || palette('textTint')};
+    position: absolute;
+    height: 100%;
+    left: 0.25rem;
+    margin-top: 1.25rem;
+  }
+
+  &:last-child::before {
+    border: 0;
+  }
+`;
+
+export const StepBullet = styled.span`
+  background-color: ${props => props.color || palette('textTint')};
```

Comment:
We can refactor this to also accept colors from the palette.
```suggestion
  background-color: ${props => palette(props.color || 'textTint')};
```

Reply:
@jxom if I try the snippet suggested, I'm not able to render the custom color (if present)....but what has worked for me is the snippet below:

 ```js
background-color: ${props =>  props.progressColor || palette('textTint')};
```

```html
<Timeline.Step progressColor="pink">...</Timeline.Step>
```

which renders as expected, a bullet with custom progressColor coloured in pink:
<img width="248" alt="screen shot 2018-12-12 at 2 46 35 pm" src="https://user-images.githubusercontent.com/41710405/49845887-c1471280-fe1c-11e8-92f3-0a65871c9dd9.png">

any ideas on what I am missing or?

Reply:
Maybe try `${props => palette(props.color || 'textTint')(props)};`

Reply:
Can you try

```js
background-color: ${props => palette(props.color || 'textTint')(props) || props.color};
```

We need it to accept colors from the palette as well.

Reply:
Can you try

```js
background-color: ${props => palette(props.color, 'textTint')(props)};
```

We need it to accept colors from the palette as well.